src/api/api.js

```diff
@@ -0,0 +1,5 @@
+module.exports = () => ({
+    gulp: {
+        running: true
+    }
+});
```

Comment:
I didnt’t get this function. What’s its intention?
Maybe we can add some comments because from the code I couldn’t understand the role of `api.js` vs `api.json`.
I got that we require `api.js` but we don’t require `api.json` anywhere.
How is the `json` structure being loaded to the server?